src/main/drivers/display.c

```diff
@@ -80,11 +80,11 @@ void displaySetXY(displayPort_t *instance, uint8_t x, uint8_t y)
     instance->posY = y;
 }
 
-int displayWrite(displayPort_t *instance, uint8_t x, uint8_t y, uint8_t attr, const char *s)
```

Comment:
`s` for string maybe, but the intention is perhaps clearer when you use `text`.  i.e. the words in the signature without all formatting is now "display write instance x y attr text" which is clearer to me.  arguably we could use 'attributes' instead of 'attr' but I'm not going that far 😄

Reply:
So long as this is consistent across all of the BF code. i.e. is "text" used to "write" to the console? or anywhere else? or is "s" used? not fussed either way - but we should be consistent.